LowPassFilter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PimaxCrystalAdvanced
+{
+    public class LowPassFilter
```

Comment:
Here is a preview with the module: https://streamable.com/l3orv9

The filtering is just weighting the last 15 Openness values together, this is to prevent jitter which makes the lids more smooth.

Reply:
I had been experimenting with the values, and 15 seemed sweet spot between responsiveness and having smoother lids.